src/domain/chains/entities/chain.entity.ts

```diff
@@ -1,9 +1,24 @@
 import { NativeCurrency } from './native.currency.entity';
+import { RpcUri } from './rpc-uri.entity';
+import { BlockExplorerUriTemplate } from './block-explorer-uri-template.entity';
+import { Theme } from './theme.entity';
+import { GasPriceOracle } from './gas-price-oracle.entity';
+import { GasPriceFixed } from './gas-price-fixed.entity';
 
 export interface Chain {
   chainId: string;
   chainName: string;
+  shortName: string;
+  rpcUri: RpcUri;
+  safeAppsRpcUri: RpcUri;
+  publicRpcUri: RpcUri;
+  blockExplorerUriTemplate: BlockExplorerUriTemplate;
+  nativeCurrency: NativeCurrency;
   transactionService: string;
   vpcTransactionService: string;
-  nativeCurrency: NativeCurrency;
+  theme: Theme;
+  gasPrice: Array<GasPriceOracle | GasPriceFixed>;
+  ensRegistryAddress: string;
```

Comment:
This is an optional value on the current client-gateway.
` pub ens_registry_address: Option<String>`

Reply:
Done in [34d40f2](https://github.com/5afe/safe-client-gateway-nest/pull/93/commits/34d40f226f67c3b006a3f1eccccfe9dc022ad746)